bokehjs/src/less/bokeh-widgets.less

```diff
@@ -1,3 +1,5 @@
-@import "vendor.less";
-@import "overrides.less";
-@import "widgets.less";
+.bk-plot {
```

Comment:
I believe this is the source of my sliders problem, the nesting isn't working as expected. My `bokeh-widgets.css` output in `bokehjs/build/css`

starts:
```css
.bk-plot {
  /* makes the font 33% larger relative to the icon container */
  /* Font Awesome uses the Unicode Private Use Area (PUA) to ensure screen
   readers do not read off random characters that represent icons */
  /*!
 * jQuery UI CSS Framework 1.11.2
 * http://jqueryui.com
 *
 * Copyright 2014 jQuery Foundation and other contributors
 * Released under the MIT license.
 * http://jquery.org/license
 *
 * http://api.jqueryui.com/category/theming/
 */

/* Layout helpers
----------------------------------*/
.bk-ui-helper-hidden {
	display: none;
}
.bk-ui-helper-hidden-accessible {
```

Note the nesting - this doesn't work in css only in less or sass. Not sure quite what the problem is.